LINE_SDK_Unity/Assets/LineSDK/Interface/LineSDKInterfaceAndroid.cs

```diff
@@ -46,20 +46,22 @@ internal static void SetupSDK(string channelId, string universalLinkURL)
             lineSdkWrapper.Call("setupSdk", parameters);
         }
 
-        internal static void Login(string scope, bool onlyWebLogin, string botPrompt, string identifier)
+        internal static void Login(string scope, bool onlyWebLogin, string botPrompt, string tokenNonce, string identifier) 
         {
-            if (!Application.isPlaying) { return; }
+          if (!Application.isPlaying) { return; }
```

Comment:
It seems like a typo.
Is it correct that the space is removed intentionally?

Reply:
Thanks! Should be fixed in 835b3a7045d0d037c08f67638d7f7f5716cab745